doc/source/visualization.rst

```diff
@@ -1269,9 +1269,10 @@ can use ``-1`` for one dimension to automatically calculate the number of rows
 or columns needed, given the other.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
Is this okwarning needed? This triggers the "No labelled objects found." warning on 0.15.2, but didn't you fix this in the other PR?